openreview/workflows/edit_invitations.py

```diff
@@ -15,6 +15,10 @@ def __init__(self, client, venue_id, update_wait_time=5000):
         self.update_date_string = "#{4/mdate} + " + str(self.update_wait_time)
 
     def save_invitation(self, invitation, replacement=None):
+        if self.get_content_value('request_form_invitation') is None:
```

Comment:
@celestemartinez I'm adding this validation here to not create any edit invitations for venues that use the current venue request form. They won't have a UI to use these invitations so it is better to not create them.